lib/tasks/job_fetch.rake

```diff
@@ -1,15 +1,11 @@
 namespace :job_fetch do
   desc "Run the stack overflow fetcher"
```

Comment:
The desc for the task should match what the task does - so this is slightly out of date now